imgee/storage.py

```diff
@@ -29,17 +29,23 @@ def get_resized_image(img, size, is_thumbnail=False):
     """
     registry = imgee.registry
```

Comment:
Also, the convention in Flask is that if you want to add something to the app, add it under `app.extensions` (a dictionary). We already have `app.extensions['lastuser']`. You could do `app.extensions['taskregistry']`. I will not recommend this however as the task registry is specific to Imgee and is not a shared extension. It's better being referenced as a direct Python import.